cmd/user-ssh-keys-agent/Dockerfile

```
@@ -12,9 +12,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM alpine:3.13
-LABEL maintainer="support@kubermatic.com"
+FROM docker.io/golang:1.16.1 as builder
+
+WORKDIR /go/src/github.com/kubermatic/kubermatic
+COPY . .
+RUN CGO_ENABLED=0 GOOS=$(go env GOOS) GOARCH=$(go env GOARCH) go build ./cmd/user-ssh-keys-agent
```

There is no point in `GOOS=$(go env GOOS) GOARCH=$(go env GOARCH)`.
```suggestion
ENV CGO_ENABLED=0
RUN go build ./cmd/user-ssh-keys-agent
```